lib/locale.t

```diff
@@ -34,6 +35,9 @@ use warnings;
 my $is_ebcdic = ord("A") == 193;
 my $os = lc $^O;
 
+# Configure now lets you build a perl that silently ignores taint features
+my $NoTaintSupport = exists($Config{taint_support}) && !$Config{taint_support};
```

Comment:
This exists check shouldn't be necessary, as this test is core-only

Reply:
While not necessary in core, I think it's worth leaving as-is, in case people look in core files to see how to check for taint support in their modules. I've documented it in perlsec, and will blog about it, but I reckon some people will just just look for examples in the perl dist.